waspc/data/Generator/templates/server/src/auth/utils.ts

```diff
@@ -137,7 +137,7 @@ async function sendEmailAndLogTimestamp(
   } catch (e) {
     rethrowError(e);  
   }
-  await emailSender.send(content);
+  emailSender.send(content);
```

Comment:
@shayneczyzewski after a  discussion with @matijaSos I've decided to remove the `await` here so we can avoid the 3 seconds loading in 99% and we'll just ignore the 1% failure cases for now